src/models/Application.py

```diff
@@ -169,6 +169,13 @@ def get_all_applications_for_user(user_id):
         return db.session.query(Application) \
             .filter((Application.assigned_to == user_id) & ((Application.score != 0) | ((Application.score == 0) & (Application.last_modified > cutoff))))
 
+    @staticmethod
+    def get_count_of_applications_for_user(user_id):
+        """Returns count of applications associated with user ID"""
+        cutoff = datetime.now() - timedelta(hours=1)
+        return db.session.query(func.count(Application.id)) \
+            .filter((Application.assigned_to == user_id) & ((Application.score != 0) | ((Application.score == 0) & (Application.last_modified > cutoff)))).scalar()
```

Comment:
can we break this up into multiple lines, it's a long condition

Reply:
Done